packages/components/src/components/auto-height-wrapper/auto-height-wrapper.jsx

```diff
@@ -24,7 +24,7 @@ const AutoHeightWrapper = props => {
     const setRef = ref => {
         if (Number.isInteger(ref?.clientHeight) && ref.clientHeight !== prev_child_client_height) {
             child_client_height_ref.current = ref.clientHeight;
-            setTimeout(updateHeight, 0);
+            setTimeout(updateHeight, 400);
```

Comment:
(question) How do we calculate the correct timeout period?

Reply:
The issue could be something to do with CSS in the `FA` screen. 
Setting the timeout as 400 could make the height adjustment laggy.

Reply:
This is minimum delay that I can do and this action allows to set correct height after loading the page.

Reply:
Does this issue happen on other screens such as `address-details`, `currency-selector`? If not, let's try to fix the issue in `financial details`.